imports/ui/components/ShoppingListButton.jsx

```diff
@@ -1,18 +1,159 @@
 import React, { Component } from "react";
-import Button from "@material-ui/core/Button";
-
+import Drawer from "@material-ui/core/Drawer";
+import List from "@material-ui/core/List";
+import Divider from "@material-ui/core/Divider";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import IconButton from "@material-ui/core/IconButton";
+import ListIcon from "@material-ui/icons/LocalGroceryStore";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import Typography from "@material-ui/core/Typography";
+import Checkbox from "@material-ui/core/Checkbox";
+import ListItemAvatar from "@material-ui/core/ListItemAvatar";
+import Avatar from "@material-ui/core/Avatar";
+import { connect } from "react-redux";
+import {
+  getShoppingListItems,
+  addNewShoppngList
+} from "../actions/AppActions.js";
+import { Meteor } from "meteor/meteor";
+import "./ShoppingListButton.css";
 
 class ShoppingListButton extends Component {
   constructor(props) {
     super(props);
+    this.state = { left: false, checked: [] };
+  }
+
+  componentDidMount() {
+    this.props.getShoppingListItems();
```

Comment:
When I log out of user1 and log in as user2, I see user1's shopping list.

Reply:
Unfortunately I cant do anything about that right now :/ I dont have access to on log out

Reply:
You need to refresh the website before viewing your own

Reply:
Resolved now